modules/Microsoft.Network/natGateways/deploy.bicep

```diff
@@ -1,7 +1,7 @@
 @description('Required. Name of the Azure Bastion resource.')
 param name string
 
-@description('Optional. The idle timeout of the nat gateway.')
+@description('Optional. The idle timeout of the NAT gateway.')
```

Comment:
General comment: this PR should also include wiki updates as per issue #2676. If we want to keep them separated, then we need to reopen the closed issue.

Reply:
My bad, I misinterpreted the issue

Reply:
Re-opened it